### DISCUSSION TITLE
DAI 2023-2024 - Practical work 3 - Trueb Guillaume and Piemontesi Gwendal

### DISCUSSION THREAD

**Discussion by tasty-orange** [heig-vd-dai-course/heig-vd-dai-course#161] [Show and tell]
We will make a teletext look a like with different subjects sending news over the network using UDP and with clients who can read the news.

**Comment by grinlemon** [comment]
![image](https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/73121793/4d1fdfb5-c83b-41a7-9288-883933837db9)

**Comment by grinlemon** [comment]
Repo link : https://github.com/truebguillaume/Teletext
Commit hash : c698b5329e0f1e6cbbd3b7109eb156ea4aa464fb
